src/studio/src/designer/backend/Languages/ini/en.ini

```diff
@@ -373,12 +373,15 @@ widget_message = Message
 
 [schema_editor]
 add = Add
-add_enum = Add value
 add_element = Add element
+add_enum = Add value
```

Comment:
Will `Add value` be sufficient for `enum`? Maybe `Add unique value`?

Reply:
Not directly tied to this change. The value is presented in this context, I think it might be fine: 
![image](https://user-images.githubusercontent.com/10416981/140477194-c3b42a60-dee5-4f10-9837-87e95766cd88.png)